src/traces/bar/attributes.js

```diff
@@ -35,8 +42,32 @@ module.exports = {
     y: scatterAttrs.y,
     y0: scatterAttrs.y0,
     dy: scatterAttrs.dy,
+
     text: scatterAttrs.text,
 
+    textposition: {
+        valType: 'enumerated',
+        role: 'info',
+        values: ['inside', 'outside', 'auto', 'none'],
+        dflt: 'none',
+        arrayOk: true,
+        description: [
+            'Specifies the location of the `textinfo`.'
```

Comment:
@n-riesco this would be a great place to add info about the inside / outside algorithm you implemented.